packages/site/src/data/graphql/libraries.ts

```diff
@@ -1,26 +1,490 @@
 import { Library } from "@framework/system/src/models/library"
 
 export const libraryTags = [
-	"state management",
-	"internationalization",
-	"framework",
-	"documentation",
-	"styling",
-	"component",
-	"logging",
-	"RxJS",
-	"performance tools",
-	"redux",
-	"firebase",
-	"data fetching",
-	"GraphQL",
-	"Utility-First CSS",
-	"CSS in JS",
-	"component library",
-	"bootstrap",
-	"forms",
-	"reactive forms",
-	"testing",
+	"Servers",
```

Comment:
It appears these `library tags` might not be flexible in this case. 

When testing it shows that libraries have specific `tags` associated for instance: `State Management, Data Fetching, Styling, Component Library, Forms, Framework, and Internationalization.`

For example `GraphQL.JS` found here can definitely be a `server` but can fit under `Data Fetching`.

![Screen Shot 2022-11-08 at 1 27 33 PM](https://user-images.githubusercontent.com/20880360/200679130-77a7fd0c-0178-42b6-be11-ca4ced0634aa.png)

Example of empty Libraries tag:
![image](https://user-images.githubusercontent.com/20880360/200679616-78828ac1-4015-4241-843e-5c071f33481f.png)

Reply:
Tbh, this gonna need a change in Framework.dev core not here, because most of these libraries are not for "Data fetching"

Reply:
Made a ticket [here](https://github.com/thisdot/framework.dev/issues/461) that I'm gonna go and tackle.